app/services/dttp/contact/create.rb

```diff
@@ -20,6 +20,7 @@ def initialize(trainee:)
       def call
         response = Client.post("/contacts", body: trainee.contact_params.to_json)
         trainee.update!(dttp_id: OdataParser.entity_id(trainee.id, response))
+        trainee.dttp_id
```

Comment:
Odd...

Is there no test coverage?

Reply:
We can actually remove this line now. We returned the `dttp_id` for manually testing. We're going to removing these classes anyway in favour of a batch request.